### PR TITLE
Add boolean type and Switch editor

### PR DESCRIPTION
**Pull request by vitPinchuk**
Resolves: #144

### CODE REVIEW

**Comment by vitPinchuk** [comment]
@asimonok @mikhail-vl ready for review

File: src/components/Table/components/CellRenderer/BooleanCellRenderer.test.tsx [hunk 1]
```diff
@@ -8,11 +8,21 @@ import { BooleanCellRenderer } from './BooleanCellRenderer';
 
 type Props = React.ComponentProps<typeof BooleanCellRenderer>;
 
+/**
+ * Mock @grafana/ui for Component to heck bgColor and color for cell
+ */
+jest.mock('@grafana/ui', () => ({
+  ...jest.requireActual('@grafana/ui'),
+  Icon: (props: any) => {
```
Comment (asimonok) [inline src/components/Table/components/CellRenderer/BooleanCellRenderer.test.tsx:16]: Please do mocks similar to another tests. Reference - `src/components/editors/TablesEditor/TablesEditor.test.tsx`

File: src/components/Table/components/CellRenderer/BooleanCellRenderer.tsx [hunk 2]
```diff
@@ -13,13 +13,35 @@ interface Props {
...
-  return <InlineSwitch disabled value={value} transparent={true} {...TEST_IDS.booleanCellRenderer.root.apply()} />;
+export const BooleanCellRenderer: React.FC<Props> = ({ value, bgColor }) => {
+  /**
+   * Theme
+   */
+  const theme = useTheme2();
+
+  return (
+    <Icon
+      {...TEST_IDS.booleanCellRenderer.icon.apply(value ? 'check-circle' : 'circle')}
```
Comment (asimonok) [inline src/components/Table/components/CellRenderer/BooleanCellRenderer.tsx:39]: I would keep the same test id and just check icon name on the element (add passing name to element in Icon mock)

File: src/constants.ts [hunk 3]
```diff
@@ -120,7 +120,7 @@ export const TEST_IDS = {
     root: createSelector('data-testid layout-cell-renderer'),
   },
   booleanCellRenderer: {
-    root: createSelector('data-testid boolean-cell-renderer'),
+    icon: createSelector((name: unknown) => String(name), 'data-testid'),
```
Comment (asimonok) [inline src/constants.ts:123]: please specify test id to prevent colisions like `data-testid boolean-cell-renderer icon`

**Review by asimonok — CHANGES_REQUESTED** [review]

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Review by mikhail-vl — APPROVED** [review]
Looks good!

**Pull request merged**